src/operation-node/or-conflict-node.ts

```diff
@@ -0,0 +1,30 @@
+import { freeze } from '../util/object-utils.js'
+import { OperationNode } from './operation-node.js'
+
+export type OrConflictNodeResolutions =
+  | 'Rollback'
+  | 'Abort'
+  | 'Fail'
+  | 'Ignore'
+  | 'Replace'
+
+export interface OrConflictNode extends OperationNode {
```

Comment:
- Is making this feature an `OperationNode` the right call? 
- And the name kinda sucks since it doesn't refer to the actual syntax used. not exactly sure what else to call it though :laughing: <br/>
SQLite calls it the `ON CONFLICT Clause` https://sqlite.org/lang_conflict.html but OnConflict is definitely off the tables since we refer to upserts by it too. So I went with OR CONFLICT.

Reply:
> Is making this feature an OperationNode the right call? I didn't manage to find docs on when to make something a node and when not to.

Some questions I ask myself when presented with this dilemma.
Is it a SQL clause? does it have complex state? do we want to allow custom compilers and plugins to visit/transform it without having to override the entire parent node?

I think it is a SQL clause, and I think we should allow visiting/transforming it without having to override the entire parent node.

Reply:
> And the name kinda sucks since it doesn't refer to the actual syntax used. not exactly sure what else to call it though

That ship has sailed a long time ago. We have stuff like `SelectModifierNode`.

SQLite [docs](https://www.sqlite.org/lang_insert.html) use `"INSERT OR action"`, so I'd lean towards `OrActionNode`, with a `node.action` property. This could possibly be reused, outside of conflicts, to append an `OR action` to a query - action could be any string really.